targeteditor/src/main/scala/explore/targeteditor/TargetBody.scala

```diff
@@ -54,14 +52,10 @@ object TargetBody {
 
   implicit val propsReuse = Reusability.derive[Props]
 
-  class Backend($ : BackendScope[Props, Unit]) {
+  class Backend {
     // Create a mutable reference
     private val aladinRef = Ref.toScalaComponent(AladinRef)
 
```

Comment:
Odd we don't need this anymore

Reply:
It was only being used in `setName`, which isn't used anymore.